# 18 Seasonal stock deviation

Does the correlation between seasonally adjusted commercial crude stocks and the Wednesday RWTC return keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: held.

WCESTUS1 level minus the train mean level for that calendar month. Month means are frozen on weeks before 2020. A later January is compared with the pre-2020 January mean, not with a mean that includes itself.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'seas-stock')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train correlation of the seasonal deviation with the Wednesday RWTC return is 0.006. Holdout correlation is 0.119. The holdout kept the train sign.
WCESTUS1 minus the frozen calendar-month mean, versus the Wednesday RWTC return
Train | 1584 | 0.006
Holdout | 348 | 0.119


## Method

The bar is sign agreement. The popular sign would be negative. The test does not switch to that story after the numbers are in.

## Result

Train correlation of the seasonal deviation with the Wednesday RWTC return is 0.006. Holdout correlation is 0.119. The holdout kept the train sign.